### PR TITLE
Chart version to 2.0.1 for CAPMC version 1.30.0

### PR DESCRIPTION
**Pull request by mbuchmann-hpe**
## Summary and Scope

Chart version to 2.0.1 for CAPMC version 1.30.0

### CODE REVIEW

File: charts/v2.0/cray-hms-capmc/Chart.yaml [hunk 1]
```diff
@@ -13,6 +13,6 @@ dependencies:
 maintainers:
   - name: Hardware Management
     url: https://github.com/orgs/Cray-HPE/teams/hardware-management
-appVersion: "1.29.0"
```
Comment (nieuwsma) [inline charts/v2.0/cray-hms-capmc/Chart.yaml:16]: LINE #4 must GO to version: 2.0.1

**Review by nieuwsma — DISMISSED** [review]
This is really close @mbuchmann-hpe just need to fix the version tag on line 4 of Chart.yaml

File: charts/v2.0/cray-hms-capmc/Chart.yaml (hunk 1 above)
Comment (mbuchmann-hpe) [reply charts/v2.0/cray-hms-capmc/Chart.yaml:16]: Updated

**Comment by mbuchmann-hpe** [comment]
> This is really close @mbuchmann-hpe just need to fix the version tag on line 4 of Chart.yaml

Update line 4

**Review by nieuwsma — APPROVED** [review]

**Pull request merged**